src/frontend/http.rs

```diff
@@ -482,8 +484,29 @@ async fn load_part(mut part: Part) -> Result<Vec<u8>, ApiError> {
     Ok(bytes)
 }
 
-// We need the allow to silence the compiler: it asks us to add warp::generic::Tuple to the first
-// parameter of the return type, but that struct is not exportable (generic is private).
+/// GET /readyz
+pub async fn health_endpoint(context: Arc<SeafowlContext>) -> Result<Response, ApiError> {
+    #[cfg(feature = "frontend-arrow-flight")]
+    if let Some(flight) = &context.config.frontend.flight {
+        // TODO: run SELECT 1 or something similar?
```

Comment:
Depends, does us being able to serve this endpoint mean the Arrow Flight frontend is initialized already? Does the Flight frontend check for connectivity on startup by loading the catalog?

Reply:
> does us being able to serve this endpoint mean the Arrow Flight frontend is initialized already?

In principle no, since there's a short period during startup when Flight frontend might not have initialized but the HTTP one has. In practice probably yes, since Flight frontend is spawned first, and there is likely a miniscule startup time for both.

> Does the Flight frontend check for connectivity on startup by loading the catalog?

No, just create a server listening fore queries. Only when one arrives does it try to load the catalog and resolve the identifiers.